src/core/server/server.h

```diff
@@ -531,7 +531,7 @@ struct grpc_server_config_fetcher {
   class ConnectionManager
       : public grpc_core::DualRefCounted<ConnectionManager> {
    public:
-    // Ownership of \a args is transfered.
+    // Ownership of \a args is transferred.
```

Comment:
This comment can actually be removed entirely.  It's obselete now that we're using a C++ type for the channel args.

Reply:
Done.